additional-databases/graphdb/README.md

```diff
@@ -0,0 +1,23 @@
+## Connecting Graph Notebook to GraphDB SPARQL Endpoint
+
+[GraphDB](https://graphdb.ontotext.com//) is a highly efficient and robust graph database with RDF and SPARQL support.
+
+For instructions on setting up and running GraphDB locally, please refer to the [GraphDB Quickstart](https://graphdb.ontotext.com/documentation/standard/quick-start-guide.html) guide.
+
+After the local setup of GraphDB is complete, the configuration to connect is the following:
+
+```
+%%graph_notebook_config
+
+{
+  "host": "localhost",
+  "port": 7200,
+  "ssl": false,
+  "sparql": {
+      "path": "repositories/<repository_id>"
```

Comment:
One thing that caused me slight confusion was that GraphDB uses two separate endpoints: one only for queries, and the second only for updates. I now understand this common to all databases running on RDF4J server. But for those who may encounter the same issues, we should clarify this, and list a second configuration that uses the update endpoint with the "/statement" prefix in the path.